components/automate-ui/src/app/modules/infra-proxy/infra-role-details/infra-role-details.component.spec.ts

```diff
@@ -102,7 +102,7 @@ describe('InfraRoleDetailsComponent', () => {
     component = fixture.componentInstance;
     element = fixture.debugElement;
     fixture.detectChanges();
-    router = TestBed.get(Router);
+    router = TestBed.inject(Router);
```

Comment:
This and the next file generated lint warnings due to method deprecation, so fixed those.